app/api.py

```diff
@@ -63,6 +63,42 @@ async def computer_assignment_rating_visualizer():
     return computer_assignment_visualizer(API.db)
 
 
+@API.post("/update/mentors")
+async def update(query: Dict, update_data: Dict):
```

Comment:
This is going to throw an error as the route and function have the same name. Instead, rename the function to update_mentors or something unique.

Reply:
Would that be the same for the other pull tequests about endpoints?

Reply:
Thank you, changes ready! 👍